format_nomad_with_env.sh

```diff
@@ -171,6 +171,12 @@ else
     environment_file="$script_directory/infrastructure/prod_env"
 fi
 
+# Temporarily set the logging config to nothing so we can see the logs
+# through nomad.
+if [[ $env == "staging" ]]; then
+    export LOGGING_CONFIG=""
```

Comment:
I assume blank defaults to DEBUG

Reply:
No, this is about where to send the logs. It defaults to using the docker driver to send them to Nomad, which we normally overwrite to send them to cloudwatch.